src/Game.java

```diff
@@ -16,9 +16,9 @@ public void start() {
             Round round = new Round(pickRandomWord(wordsToGuess), progressIndicator);
 
             round.start(scanner);
-            System.out.println("Do you want to play again? Press Y to continue or N to quit.");
+            System.out.println("Do you want to play again? Press Y to continue. Any other letter quits the game.");
             String decision = scanner.nextLine();
-            if (decision.equalsIgnoreCase("n")) {
+            if (!decision.equalsIgnoreCase("y")) {
```

Comment:
come back to this statement after reading chapter 8 of Effective Java book.

Reply:
ok, thanks! But I'm supposed to read this book only after I finish the Finnish course, so...might take a while ;-)